peering/views.py

```diff
@@ -249,6 +249,10 @@ def extra_context(self, kwargs):
             extra_context.update({"autonomous_system": autonomous_system})
         return extra_context
 
+    def setup_table_columns(self, request, permissions, table, kwargs):
+        table.columns.show("session_state")
+        super().setup_table_columns(request, permissions, table, kwargs)
+
```

Comment:
This could be done by setting the column `visible` attribute to `True`.

But I think it is a wider topic than just that. That's why I opened issue #211. I think that the column to display for tables should be left to the user own choice. By default, the visibility of the`session_state` column depends on whether or not the `NAPALM_USERNAME` and `NAPALM_PASSWORD` were set or not. But they are cases where the user is actually not using `NAPALM_PASSWORD` for instance. Maybe we should just display the column in all cases or just use a simple configuration option that could be set to `True` or `False` depending on the user actually using the session state polling feature.